pypesto/select/postprocessors.py

```diff
@@ -73,3 +74,94 @@ def save_postprocessor(
         problem.minimize_result,
         Path(output_path) / (problem.model.model_hash + ".hdf5"),
     )
+
+
+def model_id_binary_postprocessor(problem: ModelProblem):
+    """Change model IDs to binary strings.
```

Comment:
Maybe add a brieft comment that this function is intended to be used in model selection?